### ISSUE TITLE
💭 Is it possible to display the front and back cameras at the same time

### ISSUE THREAD

**Issue by baatten** [cuvent/react-native-vision-camera#2887]
### Question

Is it possible to display the front and back cameras at the same time

### What I tried

referencing previous questions:
https://github.com/mrousavy/react-native-vision-camera/issues/1564

### VisionCamera Version

4.0.4

### Additional information

- [X] I am using Expo
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for similar questions [in the issues page](https://github.com/mrousavy/react-native-vision-camera/issues?q=is%3Aopen+is%3Aissue+label%3A%22💭+question%22) as well as [in the discussions page](https://github.com/mrousavy/react-native-vision-camera/discussions) and found none.

**Comment by mrousavy** [comment]
Hey.

- On iOS, this is very likely not supported due to iOS' constraints. Per `<Camera>` component we use a single `AVCaptureSession`, and you can only have one of those active at the same time _I think_. If you want to stream data from multiple Cameras on iOS you would need to create an `AVMultiCamCaptureSession` or whatever it is called, which has an entirely different API and isn't easy to add to VisionCamera at all.
- On Android, this might be supported. Depends on the HAL implementation (aka your phone), so might work on some Samsungs, might not work on some Huaweis - I don't know.

In short; just try it, and see what happens. If it doesn't work, it doesn't work.

We could build this feature specifically for your app through my consulting agency, but that is very likely really complex.
